.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change
+
+_Select the type of your PR_
+
+- [ ] Bugfix
+- [ ] Enhancement / new feature
+
+
+### Description
+
+_Please describe your pull request_
```

Comment:
We should probably start being stricter about requiring issue numbers in PRs don't you think? In which case we should mention that as a requirement here.

Reply:
This might be a longer discussion :-). I do not think you need an issue for every PR. If it is something you can do immediately, I do not think issue is needed. Opening PR directly is fine for me.

Reply:
I know I'm sloppy about referencing (and closing) PRs when they do exist. So perhaps just a reminder to reference the issue if one exists would be enough.

Reply:
What about adding it to the checklist?